Tests/SwiftParserTest/translated/RecoveryTests.swift

```diff
@@ -767,19 +775,36 @@ final class RecoveryTests: XCTestCase {
   }
 
   func testRecovery62() {
-    assertParse(
-      """
-      enum EE 1️⃣EE<T> where T : Multi {
-        case a2️⃣ 3️⃣a
-        case b
-      }
-      """,
-      diagnostics: [
-        DiagnosticSpec(locationMarker: "1️⃣", message: "found an unexpected second identifier in enum"),
-        DiagnosticSpec(locationMarker: "2️⃣", message: "consecutive declarations on a line must be separated by ';'"),
-        DiagnosticSpec(locationMarker: "3️⃣", message: "unexpected code 'a' before enum case"),
-      ]
-    )
+    let testCases: [UInt: (fixIt: String, fixedSource: String)] = [
+      #line: (
+        "join the identifiers together",
+        """
+        enum EEEE<T> where T : Multi {
+          case a a
+          case b
+        }
+        """
+      )
+    ]
```

Comment:
Is there any benefit of the `testCases` array here if it only contains a single entry?